factions/humans.cfg

```diff
@@ -1,11 +1,11 @@
 #textdomain wesnoth-ANLEra
 [multiplayer_side]
     id=ANLEra_Humans
-    name= _ "Humans"
+    name= _ "Loyalists"
```

Comment:
This string exists in mainline, you can use `#textdomain wesnoth-multiplayer` for it.